src/server.ts

```diff
@@ -17,5 +17,12 @@ export function makeApp(db: Db): core.Express {
     response.render("index");
   });
 
+  app.get("/platform", (request, response) => {
```

Comment:
```suggestion
  app.get("/platforms", (request, response) => {
```
manque un "s" à platform